db/R/query.file.path.R

```diff
@@ -0,0 +1,19 @@
+##' @name db.query.file.path
+##' @title Get file path given id and machine
+##' @param input_id
+##' @param host_name
+##' @param con : database connection
+##' @export 
+##' 
+##' @author Betsy Cowdery 
+query.file.path <- function(input.id, host_name, con){
+  machine.host <- ifelse(host$name == "localhost",fqdn(),host_name)
+  machine = db.query(paste0("SELECT * from machines where hostname = '",machine.host,"'"),con)
+  dbfile = db.query(paste("SELECT file_name,file_path from dbfiles where container_id =",input.id," and container_type = 'Input' and machine_id =",machine$id),con)
+  path <- file.path(dbfile$file_path,dbfile$file_name)
+  if(file.exists(path)){
```

Comment:
In the short term, I'd say just disable this since it is a non-trivial check. The objective of the function is to query a file from the DB, this check isn't strictly required